api/app/signals/apps/api/v1/serializers/signal.py

```diff
@@ -163,6 +164,9 @@ class PrivateSignalSerializerDetail(HALSerializer, AddressValidationMixin):
         ]
     )
 
+    attachments = PrivateSignalAttachmentRelatedField(view_name='PrivateSignalAttachmentsViewSet', many=True,
```

Comment:
I would like to discuss the use of `view_name` here, it seems inconsistent with the documentation of DRF (https://www.django-rest-framework.org/api-guide/relations/#hyperlinkedidentityfield).

Reply:
The `PrivateSignalAttachmentRelatedField` is a subclass of the DRF `HyperlinkedRelatedField`. The view_name is required in the `HyperlinkedRelatedField` class. I needed to make a choice to either provide the view_name or overwrite the `__init__` method. Because we want a somewhat different behaviour then the default behaviour we need to fix this somewhere. So do we want overwrite the `__ini__` and move the view_name definition there?

Reply:
I it is using the class name versus the string view name defined in the URLs / by the ViewSet. The documentation see,s to only mention the latter.